frontend/src/Matching/CodeSpace.tsx

```diff
@@ -19,27 +23,57 @@ const CodeSpace = () => {
   const { socketId, difficulty } = location.state || {};
   const [socket, setSocket] = useState<Socket | null>(null);
   const [question, setQuestion] = useState<Question | null>(null);
+  const [code, setCode] = useState("console.log('hello world!');");
+  const [value, setValue] = React.useState("console.log('hello world!');");
```

Comment:
(1/2) Suggestion to use the latest code during refresh. 
Previous code will always show `console.log('hello world!');` on refresh.

```suggestion
  const [value, setValue] = React.useState(() => {
    // Retrieve the code value from localStorage or set a default value
    return localStorage.getItem('code') || "console.log('hello world!')";
  });
```

Reply:
Make sense! Thanks